altair/vegalite/v5/schema/vega-themes.json

```diff
@@ -1258,4 +1258,4 @@
       ]
     }
   }
-}
+}
```

Comment:
No idea what else could be causing changes here https://github.com/vega/altair/actions/runs/10255985015/job/28374128657?pr=3523

Reply:
@jonmmease will I need to enforce some kind of static ordering after using `vlc.get_themes()`?

Reply:
vl-convert pulls them directly from vega-themes without any sorting, so I'm not sure if this is deterministic. You should probably sort them alphabetically to be safe.

Reply:
Thanks, I'll give that a try tomorrow

Reply:
Running on Binder seems to get the correct output https://mybinder.org/v2/gh/jonmmease/vl-convert/920b52cba8f6418c77b5d349ec474bbf19f043b3?urlpath=lab%2Ftree%2Fvl-convert-python%2Fnotebooks%2Fconvert_vegalite.ipynb

Reply:
huh, I have no idea how this could be different across platforms 🤔

Reply:
In my m1 mac, for `carbong10`, I get `axis.gridColor` of `"#e0e0e0"` from vl-convert. This looks correct given:

https://github.com/vega/vega-themes/blob/af691c7f5502caac235469536a889a1a25ace969/src/theme-carbong10.ts#L3-L6

and 

https://github.com/vega/vega-themes/blob/af691c7f5502caac235469536a889a1a25ace969/src/carbongen.ts#L114

and

https://github.com/vega/vega-themes/blob/af691c7f5502caac235469536a889a1a25ace969/src/carbongen.ts#L12-L17

And somehow on windows, you're getting `"#525252"`, which corresponds to `carbong90`. I don't have a local windows machine at the moment, but I'm going to see if I can reproduce this on a windows vm on CI in a vl-convert PR.

Reply:
Unfortunately, I can't reproduce what you're seeing with a Windows runner on GitHub actions:  https://github.com/vega/vl-convert/pull/180.

Something else to try would be to try to vl-convert CLI (Download from https://github.com/vega/vl-convert/releases/tag/v1.6.0), and see what the theme definition you get there.  E.g.

```
./vl-convert cat-theme carbong10

{
  "background": "#f4f4f4",
  "arc": {
    "fill": "#6929c4"
  },
  "area": {
    "fill": "#6929c4"
  },
  "path": {
    "stroke": "#6929c4"
  },
  "rect": {
    "fill": "#6929c4"
  },
  "shape": {
    "stroke": "#6929c4"
  },
  "symbol": {
    "stroke": "#6929c4"
  },
  "circle": {
    "fill": "#6929c4"
  },
  "view": {
    "fill": "#ffffff",
    "stroke": "#ffffff"
  },
  "group": {
    "fill": "#ffffff"
  },
  "title": {
    "color": "#161616",
    "anchor": "start",
    "dy": -15,
    "fontSize": 16,
    "font": "IBM Plex Sans,system-ui,-apple-system,BlinkMacSystemFont,\".sfnstext-regular\",sans-serif",
    "fontWeight": 600
  },
  "axis": {
    "labelColor": "#525252",
    "labelFontSize": 12,
    "labelFont": "IBM Plex Sans Condensed, system-ui, -apple-system, BlinkMacSystemFont, \".SFNSText-Regular\", sans-serif",
    "labelFontWeight": 400,
    "titleColor": "#161616",
    "titleFontWeight": 600,
    "titleFontSize": 12,
    "grid": true,
    "gridColor": "#e0e0e0",
    "labelAngle": 0
  },
  "axisX": {
    "titlePadding": 10
  },
  "axisY": {
    "titlePadding": 2.5
  },
  "style": {
    "guide-label": {
      "font": "IBM Plex Sans,system-ui,-apple-system,BlinkMacSystemFont,\".sfnstext-regular\",sans-serif",
      "fill": "#525252",
      "fontWeight": 400
    },
    "guide-title": {
      "font": "IBM Plex Sans,system-ui,-apple-system,BlinkMacSystemFont,\".sfnstext-regular\",sans-serif",
      "fill": "#525252",
      "fontWeight": 400
    }
  },
  "range": {
    "category": [
      "#6929c4",
      "#1192e8",
      "#005d5d",
      "#9f1853",
      "#fa4d56",
      "#570408",
      "#198038",
      "#002d9c",
      "#ee538b",
      "#b28600",
      "#009d9a",
      "#012749",
      "#8a3800",
      "#a56eff"
    ],
    "diverging": [
      "#750e13",
      "#a2191f",
      "#da1e28",
      "#fa4d56",
      "#ff8389",
      "#ffb3b8",
      "#ffd7d9",
      "#fff1f1",
      "#e5f6ff",
      "#bae6ff",
      "#82cfff",
      "#33b1ff",
      "#1192e8",
      "#0072c3",
      "#00539a",
      "#003a6d"
    ],
    "heatmap": [
      "#f6f2ff",
      "#e8daff",
      "#d4bbff",
      "#be95ff",
      "#a56eff",
      "#8a3ffc",
      "#6929c4",
      "#491d8b",
      "#31135e",
      "#1c0f30"
    ]
  }
}
```

Reply:
@jonmmease it might be a little bit until I can test myself.

Could you check if the config I pushed is accurate prior to this PR?
- https://github.com/vega/vega-themes/pull/587

Seems possible if you're unable to repro